salt/states/boto_elb.py

```diff
@@ -339,6 +339,61 @@ def present(
     return ret
 
 
+def register_instances(name, instances, region=None, key=None, keyid=None,
+                       profile=None):
+    '''
+    Add instance/s to load balancer
+
+    add-instances:
```

Comment:
Can you add:
```
.. code-block:: yaml
```
On this line to help out the formatting of this example for the docs?